tests/func/test_config.py

```diff
@@ -98,7 +98,7 @@ def test_config_set_local(tmp_dir, dvc):
         ),
     ],
 )
-def test_config_get(tmp_dir, dvc, caplog, args, ret, msg):
+def test_config_get(tmp_dir, dvc, capsys, caplog, args, ret, msg):
```

Comment:
This might be the place where we also use `caplog`, `logger.{exception,warning,error}` don't yet have a replacements.